doc/users/prev_whats_new/whats_new_2.1.0.rst

```diff
@@ -261,11 +261,11 @@ rectangle for the size bar.
 
     fig, ax = plt.subplots(figsize=(3, 3))
 
-    bar0 = AnchoredSizeBar(ax.transData, 0.3, 'unfilled', loc=3, frameon=False,
-                           size_vertical=0.05, fill_bar=False)
+    bar0 = AnchoredSizeBar(ax.transData, 0.3, 'unfilled', loc='lower left',
```

Comment:
I wonder whether we should really change old what's new entries at all?

Reply:
In this case, I'm in favor of changing. 'lower left' was already available at that time and should have been preferred to the number. We don't change any semantics but make the example more readable.